8.x/core-concepts/cruds.md

```diff
@@ -118,204 +74,234 @@ class Todo extends BaseModel
-```
+    public function edit(Todo $todo): Factory|View|Application
+    {
 
-#### $EXACT_SEARCH_ORDER_FIELD
----
-This property configures what field the search results are sorted by.
+    }
 
-```php
-    #Search results are sorted based on creation date
-    protected static ?string $EXACT_SEARCH_ORDER_FIELD = "created_at";
-```
+    public function update(Request $request, Todo $todo): RedirectResponse
+    {
 
-### Step 3: Add Veiws
-At this stage, the view files should be created, but before that, let's add the route path and the icon file to the Larammerce project as follows.
+    }
 
-#### Add resource route
-Here, the source path for the to-do list CRUD should be added. Therefore, the following route should be written in the `routes/web.php` file.
+    public function destroy(Todo $todo): RedirectResponse
+    {
 
-##### EXAMPLE
-```php
-#TodoModel
-Route::resource("todo", "TodoController", ["as" => "admin"]);
-```
+    }
 
-:::tip NOTE
-The `web.php` file contains all system routes and including admin routes.
-:::
+    public function getModel(): ?string
+    {
+        return Todo::class;
+    }
+}
 
-####  Add an icon to the admin panel
-One of the most important parts of this tutorial is adding the task list icon in the management panel so that you can manage the tasks in it.
-It can be easily added to the admin panel by following a few simple steps.
+```
 
-Now, let's take a look at the steps.
+#### Define icon
 
-#### Step 1
----
-In the first step, the to-do list icon file (for example `icon.png`) must be added to the following path:
+A todo icon is required in the top toolbar in order to perform the crud processes.
 
-`public_html/admin_dashboard/images/icons`
+![top_toolbar](/img-cruds/01-edited.png)
 
+Search and select a `todo flat icon png` file and put it into the path `/larammerce/public_html/admin_dashboard/images/icons/`.
 
-#### Step 2
----
-After adding the icon file, the address of the icon file should be added in the path `resources/assets/sass/icons.scss`.
+Put the following code in the path `/larammerce/resources/assets/sass/icons.scss`:
 
-##### EXAMPLE
-```php
-&.icon-todo {
-    background: url("/admin_dashboard/images/icons/todo.png");
-}
+```scss{8-10}
+.h-icon {
+    background-size: 100% 100% !important;
+    ...
+    &.icon-link-shortener {
+        background: url("/admin_dashboard/images/icons/link-shortener.png");
+    }
+
+    &.icon-todo {
+        background: url("/admin_dashboard/images/icons/todo-icon.png");
+    }
+    ...
 ```
 
-To build and minify the project resource files and save the given changes, the following command should be used in the terminal.
+Run the following command in the terminal:
 
-```bash
+```bash{1}
 npm run prod
 ```
 
-#### Step 3
----
-In the third stage, the **icon** and **icon path** must be introduced to the system in the path `config/cms/appliances.php`.
+Write the code below in the path `larammerce/config/cms/appliances.php`:
```

Comment:
To introduce the appliance with the defined route and icon to the program, add the codes below in the path `larammerce/config/cms/appliances.php`.